Support/FoundationFileSystem.swift

```diff
@@ -59,9 +59,7 @@ public struct FoundationFile: File {
     try value.write(to: location)
   }
 
-  /// Append data to the file.
-  ///
-  /// Parameter value: data to be appended at the end.
+  /// Appends the bytes in `value` to the file.
   public func append(_ value: Data) throws {
```

Comment:
```suggestion
  /// Appends the bytes in `suffix` to the file.
  public func append(_ suffix: Data) throws {
```

This came from me thinking, “there's always a better name than `value`, because that word applies to every parameter.”